api/wasm/v0/cpp/proxy_wasm_externs.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Proxy-WASM ABI.
+ */
+// NOLINT(namespace-envoy)
+
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+
+//
+//  ABI calls from the host into the VM.
+//
+//  These will typically be implemented by a language specific SDK which will provide an API on top
+//  of this ABI e.g. the C++ SDK provides a proxy_wasm_api.h implementation of the API on top of
+//  this ABI.
+//
+// The Wasm VM can only access memory in the VM. Consequently, all data must be passed as integral
+// call parameters or by malloc()ing data in the VM. For consistency and to enable diverse Wasm
```

Comment:
"malloc()ing data in the host and passing it to the VM, which then must free it" right? I'm trying to reconcile this text with what you are doing in the configuration fetching API below.

Reply:
Changed this to make it clear that the Wasm code owns the allocated memory.